src/Value.js

```diff
@@ -25,6 +26,20 @@ var Value = React.createClass({
 		}
 	},
 
+  getInitialState () {
```

Comment:
Indentation seems to be wrong here, I think the whole document has 8 space of indentation.